### ISSUE TITLE
modpack_maker_list.txt contains duplicate Download URLs for different mods

### ISSUE THREAD

**Issue by Noscka** [Grokitach/Stalker_GAMMA#267]
`modpack_maker_list.txt` uses the same Download URLs for Different mods. For example `Agressor Reshade` uses the link: `https://www.moddb.com/addons/start/213337` which is correct. but then the same link gets used for `Bounty Squads Extended`, `Lsz AI tweak`, `Gotta Go Fast`, `Bizon Replacer` and more.

Is this intentional?

**Comment by JetPirate** [comment]
Same question. Looking at the history of the file, it's being quite used. Moreover, I've run a script to gather similar questionable items and it seems there are more places like that. First is the number from `/start/...`, the second is the number of occurrences:
```
222467 => 10
217186 => 7
213337 => 7
224285 => 6
200140 => 4
242234 => 3
209945 => 3
255058 => 2
243177 => 2
244792 => 2
240548 => 2
231245 => 2
219570 => 2
219923 => 2
202886 => 2
214124 => 2
211552 => 2
218608 => 2
225738 => 2
225029 => 2
210939 => 2
214696 => 2
213259 => 2
210152 => 2
208351 => 2
207498 => 2
208861 => 2
```

**Comment by Noscka** [comment]
I've looked over how everything works. And there are duplicate mods or different mods which use the same file. But after looking at the commit history, it looks like some of those download IDs get used as a placeholder, I'm guessing. So might explain some of the duplicates. Some might be accidental tho, I have a feeling that the file is manually managed

**Comment by Grokitach** [comment]
Yes this is intentional: patching shit mods / broken mods / with unharmful and small addons to preserve the addons numbering.
